templates/includes/sidebar.html

```diff
@@ -0,0 +1,17 @@
+<div class="col-sm-3">
+  <ul>
+    <li>
+      <a href="{% url 'taxi:index' %}">Home page</a>
+    </li>
+    <li>
+      <a href=#>Manufacturers</a>
+    </li>
+    <li>
+      <a href=#>Cars</a>
+    </li>
+    <li>
+      <a href=#>Drivers</a>

```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are currently placeholders (`href=#`). Consider updating these with actual URLs using the `{% url %}` template tag or remove them if they are not needed.